pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -4121,15 +4024,60 @@ func newPvcWithOwner(namespace string, name string, ownerName string, isControll
 	}
 }
 
-func newPendingVirtualMachine(name string) *virtv1.VirtualMachineInstance {
-	vmi := api.NewMinimalVMI(name)
-	vmi.UID = "1234"
-	vmi.Status.Phase = virtv1.Pending
-	setReadyCondition(vmi, k8sv1.ConditionFalse, virtv1.PodNotExistsReason)
+var defaultPendingVmiOptions = []libvmi.Option{
+	libvmi.WithNamespace(k8sv1.NamespaceDefault),
+	libvmi.WithUID("1234"),
+}
+
+var defaultPendingVmiStatusOptions = []libvmistatus.Option{
+	libvmistatus.WithPhase(virtv1.Pending),
+	readyConditionOpt(k8sv1.ConditionFalse, virtv1.PodNotExistsReason),
+}
```

Comment:
Convert these to funcs to avoid possible change of values?

Reply:
Done!